content/applications/inventory_and_mrp/manufacturing/management/split_merge.rst

```diff
@@ -0,0 +1,82 @@
+====================================
+Split and merge manufacturing orders
+====================================
+
+In Odoo *Manufacturing*, it is possible to create manufacturing orders for a single unit of an item
+or multiple units of the same item. In some cases, it may be necessary to split a manufacturing
+order that contains multiple units into two or more orders, or to merge two or more orders into a
+single order.
+
+.. important::
+   A manufacturing order can only contain one unit of a product or multiple units of a single
+   product that all use the same Bill of Materials (BoM). As a result, it is only possible to merge
+   manufacturing orders when every order contains the same product being manufactured with the same
+   BoM.
+
+Split manufacturing orders
+==========================
+
+To split a manufacturing order into multiple orders, begin by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, then select a manufacturing
+order. At the top of the page, next to the :guilabel:`New` button, the manufacturing order's
+reference number appears with a :guilabel:`⚙️ (settings)` button next to it.
+
+Click the :guilabel:`⚙️ (settings)` button to open the general settings for the manufacturing order,
+then select :guilabel:`Split`.
+
+.. image:: split_merge/settings-split.png
+   :align: center
+   :alt: The Settings and Split buttons on a manufacturing order.
+
+After selecting :guilabel:`Split`, a :guilabel:`Split production` pop-up window appears. In the
+:guilabel:`Split #` field, enter the number of manufacturing orders that the original order should
+be split into, then click outside of the field. A table appears below, with a line for each new
```

Comment:
```suggestion
be split into, then click outside of the field. A table will appear below, with a line for each new
```
To make it clear that entering a number in the *Split #** field is what makes the table appear